QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -385,9 +303,6 @@ private bool IsSupported(SecurityType securityType, Type dataType, TickType tick
             if (!_potentialUnsupportedResolutionMessageLogged)
             {
                 _potentialUnsupportedResolutionMessageLogged = true;
-                Log.Trace("DataBentoDataProvider.IsSupported(): " +
```

Comment:
You deleted a bunch of informative logging here, why? 
Statements like this "_potentialUnsupportedResolutionMessageLogged = true;" are intended to show an informative error message once. Why have the boolean if deleting the log?

Private comment was to delete the debugging logging. If there is informative logging to inform user about failures in subscription/application those will be needed